File: app/src/main/java/org/morfly/gradle/app/BindingAdapters.java
package org.morfly.gradle.app;

import androidx.databinding.BindingAdapter;
import android.widget.TextView;


public class BindingAdapters {

    @BindingAdapter("text")
    public static void text(TextView view, String text) {
        view.setText(text);
    }

    @BindingAdapter("magic_number")
    public static void customText(TextView view, int magicNumber) {
        view.setText(magicNumber + "");
    }
}

File: app/src/main/java/org/morfly/gradle/app/BindingAdapters.kt
package app.src.main.java.org.morfly.gradle.app

import androidx.databinding.BindingAdapter
import android.widget.TextView

@BindingAdapter("text")
fun text(view: TextView, text: String) {
    view.setText(text)
}

@BindingAdapter("magic_number")
fun customText(view: TextView, magicNumber: Int) {
    view.setText(magicNumber.toString() + "")
}